ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -208,29 +208,69 @@ descriptor:
+  impulse: "A force acting briefly on a body and producing a finite change of momentum."
+  irradiance: "The flux of radiant energy per unit area (normal to the direction of flow of radiant energy through a medium)."
+  length: "The measurement or extent of something from end to end."
   level: "The height of something with regard to a datum (e.g. the water level of a tank relative to the bottom)."  
   linearacceleration: "Time rate of change of linear velocity."
   linearvelocity: "Example: wind speed"
+  luminance: "The intensity of light emitted from a surface per unit area in a given direction."
+  luminousflux: "Measure of the perceived power of light."
+  luminousintensity: "Measure of the wavelength-weighted power emitted by a light source in a particular direction per unit solid angle."
+  magneticflux: "Surface integral of the normal component of the magnetic field over a surface."
+  magneticinduction: "Vector field that wdescribes the magnetic influence on moving electric charges, electric currents and magnetic materials."
+  magneticmoment: "Measure of the magnetic strength of a magnet or current-carrying coil, expressed as the torque produced when the magnet or coil is set with its axis perpendicular to unit magnetic flux density."
+  mass: "The quantity of matter which a body contains, as measured by its acceleration under a given force or by the force exerted on it by a gravitational field."
+  massflow: "Movement of fluids down a pressure or temperature gradient."
+  obscuration: "Degree of shading."
   percentage: "Measurement per hundred."
   powerfactor: "Ratio of real and apparent power."
   power: "Rate of energy consumption (assumed context is electrical and real, unless otherwise modified)."
+  powerflux: "Energy emitted, reflected, transmitted or received, per unit time."
   pressure: "Measured force per unit area within a system or process."
+  pressurechange: "Change of pressure per temperature."
   rate: "ratio of change compared to a step or time."
+  reactivepower: "Difference between the electricity supplied and the electricity converted into useful power."
   resistance: "Electrical resistance."
+  rotationalvelocity: "Vector measure of rotation rate, that refers to how fast an object rotates or revolves relative to another point."
+  specificenergy: "Energy per unit mass."
   specificenthalpy: "Measure of energy of air per unit mass."
   temperature: "Measures the temperature of media within a process or system."
-  time: "defined duration of something."
+  temperaturechangerate: "Difference in temperature per unit time."
+  temperaturedifference: "Difference of temperature."
+  temperatureintegral: "Integral of temperature over time."
+  thermalconductivity: "Measure of a material's ability to conduct heat."
+  thermalcapacity: "Amount of heat to be supplied to an object to produce a unit change in its temperature."
+  thermaltransmittance: "Rate of transfer of heat through matter. "
```

Comment:
we already have the precedent of `thermal_power_sensor` for heat transfer. is this the same thing?

Reply:
I prefer to use standard scientific terms for measurements. Thermal power describes how fast heat is produced in an energy generation system, while the terms included here are more about thermal insulation and thermal inertia.

Reply:
My concern is that this breaks precedent, and that most of these concepts can easily be constructed in a way that doesn't change precedent or readability: `thermal_powertransmittance` is equally readable and allows us to reuse the concept of `powertransmittance` when it arises from other sources (such as radiative transmittance). 

Perhaps a compromise here is to simplify to `transmittance`, which appears to have consistent use. Then, if further qualification is needed, `thermal_transmittance` can be used, and our standard can be maintained.